src/SWP/Bundle/CoreBundle/Form/Type/ThemeLogoUploadType.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2018 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2018 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Form\Type;
+
+use SWP\Bundle\CoreBundle\Theme\Model\Theme;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\File;
+
+final class ThemeLogoUploadType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('logo', FileType::class, [
+                'required' => true,
+                'constraints' => [
+                    new File([
+                        'maxSize' => '5120k',
```

Comment:
why you limit it here? Imo it should be handled by php/webserwer settings

Reply:
The same limit is set for theme upload zip file so I used the same.

Reply:
5mb, ok let's leave it like this.